src/azure-cli/azure/cli/command_modules/keyvault/_params.py

```diff
@@ -399,6 +399,10 @@ class CLIJsonWebKeyType(str, Enum):
     for scope in ['list', 'list-deleted', 'list-versions']:
         with self.argument_context('keyvault key {}'.format(scope)) as c:
             c.argument('maxresults', options_list=['--maxresults'], type=int)
+
+    with self.argument_context('keyvault key list') as c:
+        c.extra('include_managed', arg_type=get_three_state_flag(), default=False,
+                help='Include managed keys. Default: false')
```

Comment:
If you remove ` Default: false` in help text, will a same message be appended by `default=False`?

Reply:
@fengzhou-msft No, so I added `Default: false` to reduce unexpected behavior.